.github/workflows/instant_benchmark.yml

```diff
@@ -95,6 +110,19 @@ jobs:
           --job ${{ matrix.job }} --instance ${{ github.event.inputs.instance }} \
           --container deepjavalibrary/djl-serving:${{ github.event.inputs.container }}
           bash instant_benchmark.sh
+      - name: Configure AWS Credentials
```

Comment:
you don't have to assume role, the default role carried inside of hosted runner should have db write permission

Reply:
those permission only exposed to EC2 launched-runner

Reply:
I would like to leave it like that for now. In the future, we probably need to look at what role these runners are getting to help lock down our security. It should probably be either a new role or the same one I am explicitly assuming here with minimal permissions as used in our actions. Maybe even a blank role that has no permissions at all which would require this explicit role assumption to indicate the escalating permissions